kuksa_databroker/lib/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+#********************************************************************************
+# Copyright (c) 2022, 2023 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License 2.0 which is available at
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# SPDX-License-Identifier: Apache-2.0
+#*******************************************************************************/
+
+[package]
+name = "common"
+version = "0.1.0"
+authors = ["Robert Bosch GmbH"]
```

Comment:
As part of the release I replaced most instances of this line with "Eclipse KUKSA Project" after a discussion with Sebastian. Could be good to do that also for the new files